src/Signer/Hmac/Blake2b.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Signer\Hmac;
```

Comment:
Blake2 is a MAC not HMAC, let's move it to `Lcobucci\JWT\Signer`?

Reply:
Blake *is* an hash function, so it is a HMAC, doesn't it? :thinking:

Reply:
AFAIK, HMAC is a MAC algo that is backed up by a hashing algo (ergo hmac-sha*). There are even implementations of hmac-blake2.

Blake2 has a MAC mode (when the key is provided).

Reply:
Heh, definitions are indeed sharper then reality.

Right, this is not a pure HMAC impl (it's better since it pads the key, and so on) even though it's practically the same impl

Reply:
Handled :+1: